src/envs/behavior_options.py

```diff
@@ -651,7 +676,7 @@ def place_obj_plan(
     original_state: int,
     place_rel_pos: Array,
     rng: Optional[Generator] = None,
-) -> List[List[float]]:
+) -> Optional[List[List[float]]]:
```

Comment:
why this change?

Reply:
turns out it actually was always optional and this was just a bug (caught thanks to the refactor)!

Reply:
(it's optional because the plan might be None)